root/src/components/aboutUs/MeetOurTeam/MeetOurTeam.tsx

```diff
@@ -128,7 +118,7 @@ const MemberCard = ({
         </div>
       </div>
       {/* Name below the card*/}
-      <p className="text-center font-bold text-3xl text-slate-800 py-1 dark:text-dark-text1 group-hover:hidden">
+      <p className="text-center font-bold text-2xl sm:text-3xl text-slate-800 px-3 py-1.5 dark:text-dark-text1 group-hover:hidden">
```

Comment:
Lets use our colors instead of tailwinds

Reply:
Got it!